ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3771,4 +3772,31 @@ VSZ:
   uses:
   - zone_air_temperature_sensor
   - supply_air_damper_percentage_sensor
-  - supply_air_flowrate_sensor
\ No newline at end of file
+  - supply_air_flowrate_sensor
+
+CHCS:
+  description: "Chllled water isolation valve control."
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  uses:
+  - chilled_water_isolation_valve_percentage_command
+  - chilled_water_isolation_valve_percentage_sensor
+
+CWCS:
+  description: "Condensing water isolation valve control."
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  uses:
+  - condensing_water_valve_percentage_command
+  - condensing_water_valve_percentage_sensor
+
+PFSC:
```

Comment:
This should be PMP_SS_VSC.

Reply:
changed to FSC with speed_frequency_command and speed_frequency_sensor.